src/unstract/adapters/llm/azure_open_ai/src/azure_open_ai.py

```diff
@@ -75,3 +73,7 @@ def test_connection(self) -> bool:
         llm = self.get_llm_instance()
         test_result: bool = LLMHelper.test_llm_instance(llm=llm)
         return test_result
+
+    def get_context_window_size(self) -> int:
```

Comment:
Is it allways return zero?

Reply:
Thank you for catching this @muhammad-ali-e